lib/src/test/kotlin/starknet/provider/ProviderTest.kt

```diff
@@ -588,7 +587,7 @@ class ProviderTest {
         val payload =
             DeclareTransactionPayload(
                 ContractDefinition(contents),
-                DECLARE_SENDER_ADDRESS,
+                Felt.ONE, // Declare tx version 0 has a sender address of 0x1
```

Comment:
I don't like replacing properly named constant with a comment. Code > comments.

Reply:
Generally I agree, but this constant only made sense in v0 declare transactions. Ideally we'll update our tests to only create declare v1 and this will be replaced entirely.